sympy/geometry/plane.py

```diff
@@ -786,12 +799,93 @@ def random_point(self, seed=None):
 
         Point3D
 
+        Examples
+        ========
+
+        >>> from sympy import Plane
+        >>> p = Plane((1, 0, 0), normal_vector=(0, 1, 0))
+        >>> r = p.random_point(seed=42)  # seed value is optional
+        >>> r.n(3)
+        Point3D(2.29, 0, -1.35)
+
+        The random point can be moved to lie on the circle of radius
+        1 centered on p1:
+
+        >>> c = p.p1 + (r - p.p1).unit
+        >>> c.distance(p.p1).equals(1)
+        True
         """
         import random
         if seed is not None:
             rng = random.Random(seed)
         else:
             rng = random
         u, v = Dummy('u'), Dummy('v')
-        params = {u: Rational(rng.random()), v: Rational(rng.random())}
+        params = {
+            u: 2*Rational(rng.gauss(0, 1)) - 1,
```

Comment:
This should be simply `Rational(rng.gauss(0, 1))` (and same for v). The transform with 2*(...)-1 is used to make the uniform distribution on [0, 1] into a uniform distribution on [-1, 1], thus a symmetric one. But the standard Gaussian distribution gauss(0, 1) is already symmetric: its density is bell curve with the peak at 0. Shifting it makes it non-symmetric.